packages/wrangler/src/pages/upload.tsx

```diff
@@ -131,7 +132,7 @@ export const upload = async (
 					setTimeout(resolvePromise, Math.pow(2, attempts++) * 1000)
 				);
 
-				if ((e as { code: number }).code === 8000013) {
+				if ((e as { code: number }).code === ApiErrorCodes.UNAUTHORIZED) {
```

Comment:
Unrelated to this PR, but looks like maybe we should add `|| isJwtExpired(jwt)` here too?

Reply:
I was wondering abt that as well